IBAnimatable/Chainable.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  Chainable.swift
```

Comment:
Should we name it `AnimationChainable` to be more coherent?
You can also drop these lines to follow IBAnimatable convention!